common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,121 @@
+"""OLI analytics service event tracker backend."""
```

Comment:
The code is more easily modified in the future (smaller diff) by
treating this as a multi-line docstring from inception.

```python
"""
OLI analytics service event tracker backend
"""
```

This way, if changes are later made to the body of the docstring
(additional information added), it doesn't look like the entire
docstring is changing; it'll just look like lines were added.


```python
"""
OLI analytics service event tracker backend
"""
```
changed to
```python
"""
OLI analytics service event tracker backend

This is used to hand analytics data over to Google blah blah blah.
"""
```
diffs like
```diff
+
+This is used to hand analytics data over to Google blah blah blah.
```

compared to


```python
"""OLI analytics service event tracker backend"""
```
changed to
```python
"""
OLI analytics service event tracker backend

This is used to hand analytics data over to Google blah blah blah.
"""
```
diffs like
```diff
-"""OLI analytics service event tracker backend"""
+"""
+OLI analytics service event tracker backend
+
+This is used to hand analytics data over to Google blah blah blah.
+"""
```

Any time an expression has multiple ways to be written, we should opt
for the more future-proof version.